python/cudf/cudf/core/dtypes.py

```diff
@@ -268,6 +268,10 @@ def __init__(self, precision, scale=0):
         self._validate(precision, scale)
         self._typ = pa.decimal128(precision, scale)
 
+    @property
+    def str(self):
+        return "decimal"
```

Comment:
PyArrow does this as `decimal(precision, scale)` should we do the same? Otherwise we won't be able to round trip this as expected.

Reply:
Are you using `str(pa.decimal128(1,0))` to get that? `.str` does something different from`str()` in the numpy world apparently

```
>>> dt = np.dtype('int64')
>>> dt.str
'<i8'
>>> str(dt)
'int64'
```

Reply:
Just an FYI, `dtype.str` is specifically designed for compatibility with the `__array_interface__` protocol. The meaning is documented [here](https://numpy.org/doc/stable/reference/arrays.dtypes.html) (or [here](https://numpy.org/doc/stable/reference/arrays.interface.html#python-side)]). In this case, `<i8` indicates a little-endian 8-byte integer type.